components/data-apps/index.md

```diff
@@ -73,10 +109,57 @@ For writing data back to Keboola Project Storage, use our [Keboola Storage Pytho
 See the [examples](#Examples) below for usage of the Keboola Storage Python Client.
 
 ## Deployment and App Management
-{% include warning.html content="Once the data app is deployed, its URL will be publicly available! Keboola does not provide authorization to data apps out of the box yet." %}
+{% include warning.html content="Once the data app is deployed, its URL will be publicly available! Keboola does provide two authorization methods." %}
 
 ### Authorization
-We recommend incorporating some sort of authorization into your app—for example, the Streamlit authenticator. Check out the [Streamlit authenticator tutorial](https://blog.streamlit.io/streamlit-authenticator-part-1-adding-an-authentication-component-to-your-app/) or take a look at [our example](https://github.com/KB-PS/mkt-bi-ocr/blob/master/Select_Invoices.py).
+We recommend to use authorization methods provided by Keboola.
+
+{: .image-popup}
+![Code - code](/components/data-apps/data_app-authentication.png)
+
+1. **Simple Authorization**: This method allows you to authenticate one user using a username and password.
+
+{: .image-popup}
+![Code - code](/components/data-apps/data_app-authentication-basic.png)
+
+2. **OIDC (OpenID Connect) Authorization**: This method enables users to log in to your app using your SSO providers without the need of having their own existing Keboola accounts. 
```

Comment:
> without the need of having their own existing Keboola accounts

Tohle bych sem nedaval. Nebo to vytahnul vis - plati to totiz pro vsechny autorizacni metody ne jenom pro tuhle (OIDC).